documentation/asciidoc/accessories/iqaudio/advanced.adoc

```diff
@@ -1,36 +1,35 @@
 == Hardware information
 
-=== GPIO usage / Pin headers
+=== GPIO usage 
 
-Raspberry Pi Audio boards take advantage of a number of Raspberry Pi Pi signals in
-order to operate successfully. Some of these signals are solely for the use of the board and
+Raspberry Pi audio boards take advantage of a number of pins on the GPIO expander in
```

Comment:
You've got other pending CE's which have `Raspberry Pi Audio Boards` all capitalised - see #2639 and #2641.

`GPIO expander` is a type of chip. (Although [this](https://www.raspberrypi.com/news/gpio-expander/) confused things by reusing the term for a piece of software). The concensus thus far in the repo tends towards `GPIO header` as the term used to describe the 40 pin header on Raspberry Pi boards.

Reply:
Won't fix